Pi3BoardPkg/Drivers/MmcDxe/MmcBlockIo.c

```diff
@@ -514,6 +514,7 @@ EFI_STATUS InitializeMmcDevice(
 
     if (MmcHostInstance->CardInfo.CardType == MMC_CARD) {
         // Fetch ECSD
+        CmdArg = MmcHostInstance->CardInfo.RCA << 16;

```

Comment:
CMD8 argument for eMMC is entirely stuff bits. Should just set CmdArg to 0 for clarity.